diff --git a/dist/generation/generate-dotpkg.js b/dist/generation/generate-dotpkg.js
index 1347945e8bdc05e4b10099576301023b63ba9cd9..4e4bc29f8f2b548649c6d18d72f6521cf877dc49 100644
--- a/dist/generation/generate-dotpkg.js
+++ b/dist/generation/generate-dotpkg.js
@@ -50,7 +50,7 @@ const writeFilesForCache = ({ config, cache, schemaDef, targetPath, generationOp
         (nodeVersionMajor === 20 && nodeVersionMinor >= 10);
     const importAttributeStatement = needsJsonimportAttributeStatement
         ? ` with { type: 'json' }`
-        : ` assert { type: 'json' }`;
+        : ` with { type: 'json' }`;
     const typeNameField = generationOptions.options.fieldOptions.typeFieldName;
     const dataBarrelFiles = documentDefs.map((docDef) => ({
         content: makeDataExportFile({
diff --git a/src/generation/generate-dotpkg.ts b/src/generation/generate-dotpkg.ts
index 14cbe0c258fcbffd525ac211e4c2ba6cc994c9a9..f3259116642dab7825d3b92284ea358ab6cc8d1d 100644
--- a/src/generation/generate-dotpkg.ts
+++ b/src/generation/generate-dotpkg.ts
@@ -199,7 +199,7 @@ const writeFilesForCache = ({
         (nodeVersionMajor === 20 && nodeVersionMinor >= 10)
       const importAttributeStatement = needsJsonimportAttributeStatement
         ? ` with { type: 'json' }`
-        : ` assert { type: 'json' }`
+        : ` with { type: 'json' }`
 
       const typeNameField = generationOptions.options.fieldOptions.typeFieldName
       const dataBarrelFiles = documentDefs.map((docDef) => ({
